docroot/profiles/custom/cgov_site/themes/custom/ncids_trans/front-end/src/legacy/styles/_general.scss

```diff
@@ -77,6 +77,35 @@ h2.related-resources__heading {
   font-style: normal;
 }
 /** */
+// A11Y Updated Video Title Heading
+span.video-title__heading {
+  display: block;
+  font-family: Poppins, Helvetica Neue, Helvetica, Roboto, Arial, sans-serif;
+  font-size: 1.125em;
+	font-weight: normal;
+  margin-top: 1em;
+  margin-bottom: 0.5em;
+	line-height: 24px;
+	outline: none;
+}
+// A11Y Updated IC Carousel Heading
+span.ic-carousel-title__heading{
+  display:block;
+  line-height: 1.333em;
+  font-family: Poppins, Helvetica Neue, Helvetica, Roboto, Arial, sans-serif;
+  font-size: 1.125em;
+  font-weight: bold;
+  margin-top: 1em;
+  margin-bottom: 0.5em;
+}
+// A11Y Updated YT Video Carousel Heading
+figcaption.yt-video-carousel__heading{
+  font-family: Poppins, Helvetica Neue, Helvetica, Roboto, Arial, sans-serif;
+  line-height: 1.333em;
+  font-size: 1.125em;
+  margin-top: 1em;
+  margin-bottom: 0.5em;
+}
```

Comment:
Please make a copy of https://github.com/NCIOCPL/cgov-digital-platform/blob/develop/docroot/profiles/custom/cgov_site/themes/custom/cgov/src/styles/_image-video-carousel.scss and put it in the ncids_trans legacy styles. This is because this partial defines both the image and video carousels.

Also this should use USWDS mixins too.